src/Common/src/TypeSystem/Interop/IL/NativeStructType.cs

```diff
@@ -37,6 +37,14 @@ public override string Namespace
             }
         }
 
+        public override Instantiation Instantiation
```

Comment:
Incorrectly resolved conflicts?

Reply:
Thanks. I'm also going to have to do some refactoring here when I get the ckfinite one right - use the existing helpers..

Reply:
This has gone now.